pyproject.toml

```diff
@@ -11,7 +11,7 @@ authors = [
 ]
 description = "A language-agnostic LSP client in Python, with a library interface. Intended to be used to build applications around language servers. Currently multilspy supports language servers for Python, Rust, Java, Go, JavaScript and C#. Originally appeared as part of Monitor-Guided Decoding (https://github.com/microsoft/monitors4codegen)"
 readme = "README.md"
-requires-python = ">=3.7"
+requires-python = ">=3.8, <4.0"
```

Comment:
Can you kindly undo the "<4.0" change?

Reply:
as I wrote if I do it poetry lock command failed with
For jedi-language-server, a possible solution would be to set the `python` property to ">=3.8,<4.0"